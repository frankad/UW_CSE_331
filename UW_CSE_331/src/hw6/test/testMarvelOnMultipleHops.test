# Find a path with multiple hops using the full Marvel dataset.

# build graph
LoadGraph marvel marvel.tsv

# test the findPathBFS
FindPath marvel GORILLA-MAN MOJO