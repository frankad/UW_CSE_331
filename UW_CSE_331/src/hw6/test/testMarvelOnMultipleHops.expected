# Find a path with multiple hops using the full Marvel dataset.

# build graph
loaded graph marvel

# test the findPathBFS
path from GORILLA-MAN to MOJO:
GORILLA-MAN to 3-D MAN/CHARLES CHAN via AVF 4 
3-D MAN/CHARLES CHAN to ANGEL/WARREN KENNETH via COC 1
ANGEL/WARREN KENNETH to MOJO via M/CP 89/4